src/main/java/core/basesyntax/exception/ValidationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class ValidationException extends Exception {

```

Comment:
Consider extending RuntimeException instead of Exception, as it does not require the calling method to handle the exception explicitly.